src/mono/sample/wasm/browser/main.js

```diff
@@ -22,6 +24,9 @@ async function main() {
             return {
                 disableDotnet6Compatibility: true,
                 configSrc: "./mono-config.json",
+                locateFile: (path, prefix) => {
+                    return window.location.protocol + "//" + location.host + "/" + prefix + path;
```

Comment:
```suggestion
                    return window.location.origin + "/" + prefix + path;
```

nit: It involves less concatenation + includes current port (if any non default)